src/test/java/uk/gov/pay/api/exception/mapper/CreateAgreementExceptionMapperTest.java

```diff
@@ -36,4 +37,17 @@ void testExceptionMapping() {
                 is(expectedError.getDescription()));
         assertThat(returnedError.getCode(), is(expectedError.getCode()));
     }
+    
+    @Test
+    void testExceptionMappingForRecurringCardPaymentsNotAllowed() {
+        when(mockResponse.readEntity(ConnectorErrorResponse.class))
+                .thenReturn(new ConnectorErrorResponse(ErrorIdentifier.valueOf("RECURRING_CARD_PAYMENTS_NOT_ALLOWED"), null, null));
```

Comment:
Is there a good reason we lookup the enum values by a String value here and for the `RequestError.Code` rather than just using the enum constants directly? 

Looks like the pattern of doing this already exists for other tests, but I can't see why we'd do this. Maybe this was copied from a parameterised test at some point where it was necessary to look up from a string value?

Reply:
no idea - yes I followed what I saw elsewhere. Will adjust.

Reply:
amended to refer to enum constants as suggested